docs/network-flow-visibility.md

```diff
@@ -220,8 +220,9 @@ We support following connection metrics as Prometheus metrics that are exposed
 through [Antrea Agent apiserver endpoint](prometheus-integration.md):
 `antrea_agent_conntrack_total_connection_count`,
 `antrea_agent_conntrack_antrea_connection_count`,
-`antrea_agent_denied_connection_count` and
-`antrea_agent_conntrack_max_connection_count`
+`antrea_agent_denied_connection_count`,
+`antrea_agent_conntrack_max_connection_count`, and
+`antrea_agent_reconnection_to_flow_aggregator_count`
```

Comment:
I think `antrea_agent_flow_collector_reconnection_count` is a better name:
- in the yaml, we refer to it as the "flow collector"; IIRC it doesn't _have_ to be the FlowAggregator
- name is more similar to existing metrics IMO

Reply:
Makes sense to me. Updated. Thanks!